py_src/yolov4/tflite/__init__.py

```diff
@@ -138,7 +138,7 @@ def predict(self, frame: np.ndarray):
         self.interpreter.set_tensor(self.input_index, image_data)
         self.interpreter.invoke()
         candidates = [
-            self.interpreter.get_tensor(self.output_index[i]) for i in range(3)
+            self.interpreter.get_tensor(self.output_index[i]) for i in range(len(self.output_index))
```

Comment:
```python
        candidates = [
            self.interpreter.get_tensor(index) for index in self.output_index
        ]
```

Reply:
Yeah, that's much better. I'm a python noob. Thanks!